test/hummingbot/connector/gateway/amm/debug_gateway_cancel.py

```diff
@@ -38,7 +38,7 @@ class GatewayCancelDataCollector:
     def __init__(self):
         self._clock: Clock = Clock(ClockMode.REALTIME)
         self._connector: GatewayEVMAMM = GatewayEVMAMM(
-            "uniswap",
+            "amm",
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed